packages/frontend-2/components/tour/Comment.vue

```diff
@@ -83,6 +99,7 @@ defineEmits(['skip', 'previous', 'next'])
 const props = defineProps<{
   index: number
   item: SlideshowItem
+  hasAddedOverlay: boolean
```

Comment:
Should this be a more generic `disableNext`? Because `hasAddedOverlay` is only relevant voor 1 specific TourComment